sync.md

```diff
@@ -1,284 +0,0 @@
-# Chain Syncing
-
```

Comment:
where does all the information from this file live now? seems to be a lot if it was just dropped

Reply:
Well, 230 out of the 300 lines were actually code, describing interfaces and even implementation specifics, that shouldn't be in a spec in the first place, as they are left to the implementers (**if at all** they could be put in an appendix of a reference implementation). Same goes for the operating in two modes - if you want to understand it as such, the implementer may, but that isn't strictly necessary. 

Rather I put a brief [syncing description](https://github.com/filecoin-project/specs/pull/295/files/62fa09f0648c12b04878c5f06d2bef8dbf48cb53#diff-f9719a6664c477d6b213f13bc09ac677R34) inspired by this file about sync in the networking.md but cut it down to the essentials and constraints - e.g. while importing, you must not mine. I found most other information in this file to be redundant to what is already said at other, more appropriate places. But please let me know if I missed anything.

Reply:
thanks, will go through it again